components/StaticLandingPage.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import ReactNative, {Button, NativeEventEmitter, Platform} from 'react-native';
+import client, { Color } from '@doubledutch/rn-client'
+import Footer from './Footer'
+const { TouchableOpacity, TouchableHighlight, Text, View, Image, WebView, Dimensions, Linking } = ReactNative
+
+export default class StaticLandingPage extends Component {
```

Comment:
It seems like this is very similar to `LandingPage`.  We should compose components to avoid the duplication.